scripts/add_export.py

```diff
@@ -77,6 +77,7 @@ def add_export(n, hydrogen_buses_ports, export_profile):
         "Bus",
         "H2 export bus",
         carrier="H2",
+        location="World",  # TODO: revise
```

Comment:
I see the purpose, does "world" make sense?

Reply:
Other recommedation?
Earth?

I would avoid something customizable that is likely an unnecessary complication, if you agree too

Reply:
Earth sounds great, let's do that

Reply:
Done